src/simplewallet/simplewallet.cpp

```diff
@@ -3691,6 +3805,17 @@ bool simple_wallet::show_transfers(const std::vector<std::string> &args_)
     }
   }
 
+  // subaddr_index
+  std::set<uint32_t> subaddr_indices;
+  if (local_args.size() > 0 && local_args[0].substr(0, 6) == "index=")
+  {
+    if (!parse_subaddress_indices(local_args[0], subaddr_indices))
+      return true;
+    in = pool = true;
+    out = pending = failed = false;
```

Comment:
Now the revised version looks like this:
```
[wallet/0 9ufcBz]: show_transfers out index=1,2
  951375    out       2017-07-11      15.000000000000 3fd18216c9aa35d3afe69c1370cebb408e2fb9db519156c4332f5decf466d251 0000000000000000 0.023295440000  0,1,2 - 
  953215    out       2017-07-13       0.000000000000 e22759429b094a50b6b50eee8bbe461a1f0863428b30225babc80017a0ebb6de 0000000000000000 0.021556080000  1 - 
  953228    out       2017-07-13       0.000000000000 2d5d3b8b34774a8a2636912b4dc870261db4eda2ab49ebe5a876868954e96960 0000000000000000 0.023213680000  0,1 - 
  953373    out       2017-07-14       0.000000000000 addc8abf6d27337f82da8f4959cf7d4d25cef891c0fe92edd9c933ba11b48463 0000000000000000 0.021549320000  1 - 
  953402    out       2017-07-14       0.000000000000 e896a2ff5a7ba5adc06851d4a775c0e6816b45cde685db03ac6a5b85b0f8f7f9 0000000000000000 0.021548280000  1 - 
  953430    out       2017-07-14       0.000000000000 07031c6fb8eb8d376fa39e125085036e056e7122d92c01fd5353c274bc88c65e 0000000000000000 0.021547240000  1 - 
  953432    out       2017-07-14       0.000000000000 db1da922f4b1cd51da3efe431ff5a4627274a800ec21fd816982a3f49b027b26 0000000000000000 0.021546720000  2 - 
  953443    out       2017-07-14       0.000000000000 c3c30203f375a7053f147dc4d738b64fff6d1549d8b038824e021753aa9b6d24 0000000000000000 0.023204160000  0,1 - 
  953585    out       2017-07-14       0.000000000000 3cfa8d5278c6b9b8ef5ec946fc120b35284ef2454402f25b479591f0c72af396 0000000000000000 0.023197440000  0,1 - 
  953589    out       2017-07-14       0.000000000000 12c5c03cda3c95f314622d32532f5b7dbbf0442eed21e1b29c661fe2685a0b0d 0000000000000000 0.021540480000  2 - 
  953611    out       2017-07-14       0.000000000000 ccebd0b3b6aafcb8ce5db30cd1e8c5ad90ecfdf5697cea47d6b37d2ec8af5fab 0000000000000000 0.021539440000  2 - 
  955168    out      01:28:12 PM      37.000000000000 fd6490a1491b796f3c18dfb1538558d5671e652dd141df3b0ec340026e40f7de 0000000000000000 0.047906840000  0,1,2,3 - 
```
Notice how some outgoing payments are associated with multiple subaddress indices (comma delimited).